packages/runtime/src/runtime.ts

```diff
@@ -62,55 +63,67 @@ export class Runtime {
     return a;
   }
 
+  /**
+   * asserts if account address is defined
+   * @param addr account address
+   * @param line line number in TEAL file
+   * Note: if user is accessing this function directly through runtime,
+   * the line number is unknown
+   */
+  assertAddressDefined (addr: string | undefined, line?: number): string {
+    if (addr === undefined) {
+      throw new TealError(ERRORS.TEAL.ACCOUNT_DOES_NOT_EXIST, { line: line });
+    }
+    return addr;
+  }
+
   /**
    * asserts if application exists in state
+   * @param app application
    * @param appId application index
    * @param line line number in TEAL file
    * Note: if user is accessing this function directly through runtime,
    * the line number is unknown
    */
-  assertAppDefined (appId: number, line?: number): SSCAttributesM {
+  assertAppDefined (app: SSCAttributesM | undefined, appId: number, line?: number): SSCAttributesM {
```

Comment:
```suggestion
  assertAppDefined (app?: SSCAttributesM, appId: number, line?: number): SSCAttributesM {
```
We are using `| undefined` in 1st arg and `?` in 3rd

Reply:
yes